addons/mail/static/src/discuss/call/common/call.js

```diff
@@ -64,6 +73,12 @@ export class Call extends Component {
             this.resizeObserver.disconnect();
             browser.clearTimeout(this.overlayTimeout);
         });
+        useEffect(
+            () => {
+                this.env.discussCall.collapsed = this.state.collapsed;
+            },
+            () => [this.state.collapsed]
+        );
```

Comment:
Instead of using this `useEffect` you can try assigning `discussCall.collapsed` with `state.collapsed`.

Reply:
Tried it, it doesn't work as expected.